var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -54,6 +54,13 @@ def mpi_libs(self):
             libraries, root=self.prefix.lib64, shared=True, recurse=True
         )
 
+    @property
+    def mpi_headers(self):
+        # recurse from self.prefix will find too many things for all the
+        # supported sub-architectures like 'mic'
+        return find_headers(
+            'mpi', root='{0}64'.format(self.prefix.include), recurse=False)
```

Comment:
Never seen an `include64` before, but Intel does some weird things. If you want, you can add it to `lib/spack/spack/util/prefix.py` so that you can use `self.prefix.include64`.